dev-util/acr/acr-1.6.2.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="An autoconf replacement"
+DESCRIPTION="ACR tries to replace autoconf functionality generating a \
+full-compatible 'configure' script (runtime flags). But using shell-script \
+instead of m4. This means that ACR is faster, smaller and easy to use."
+HOMEPAGE="https://github.com/radare/acr"
+COPYRIGHT="2005-2018 pancake"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/radare/acr/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="12b2f5ceb4da4507fba5242fe5596b8e25916478709f1d6189cbc016b01e6661"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64 ?arm"
```

Comment:
```diff
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64 ?arm"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64 ?arm"
```
The recipe passes all tests on both x86_gcc2 and x86_64. (I did not test on x86 but I guess we would have the same results.)

So, I think we should drop the old recipe, `acr-0.9.9.recipe`.
But since this would to be shown as:
```diff
--- a/dev-util/acr/acr-0.9.9.recipe
+++ /dev/null
@@ -1,61 +0,0 @@
```
and
```diff
diff --git a/dev-util/acr/acr-1.6.2.recipe b/dev-util/acr/acr-1.6.2.recipe
new file mode 100644
--- /dev/null
+++ b/dev-util/acr/acr-1.6.2.recipe
```
we would not have a nice `git log -p`.

But there's something we can do to turn this into a nicer `git log -p` (i.e. with a similarity index >= 50%):
```diff
diff --git a/dev-util/acr/acr-0.9.9.recipe b/dev-util/acr/acr-1.6.2.recipe
similarity index 53%
rename from dev-util/acr/acr-0.9.9.recipe
rename to dev-util/acr/acr-1.6.2.recipe
index b35a5f777..23187a4b3 100644
--- a/dev-util/acr/acr-0.9.9.recipe
+++ b/dev-util/acr/acr-1.6.2.recipe
```

The first step is to change `acr-0.9.9.recipe` a little to make it look more similar to `acr-1.6.2.recipe`.
(I'll do that step in a while.)
The second step is to rebase your acr branch against master (after the first step is done.)
(I can do this in your PR if you wish.)